azkaban-hadoop-security-plugin/src/main/java/azkaban/security/AbstractHadoopSecurityManager.java

```diff
@@ -618,6 +628,16 @@ public void cancelTokens(final File tokenFile, final String userToProxy, final L
     }
   }
 
+  @Override
+  public Credentials getTokens(File tokenFile, Logger logger)
+      throws HadoopSecurityManagerException {
+    try {
+      return Credentials.readTokenStorageFile(new Path(tokenFile.toURI()), this.conf);
```

Comment:
we seem to be not using the logger instance here. did we intent to log something?

Reply:
Good call, just follow the pattern for other HSM methods